app.py

```diff
@@ -6,14 +6,31 @@
 Description: Entry point of the app, contains API implementation.
 """
 
-from fastapi import FastAPI, Header
+from fastapi import FastAPI, Header,File,UploadFile
 from pydantic import BaseModel
 from utils.pinecone_vectorstore import build_pinecone_index
 from utils.chat import get_chat_response
 from auth import is_authorized_request
+import shutil
+from pathlib import Path
```

Comment:
what is the use of this module?

Reply:
using shutil to create a file from file object that we are recieving from Front end .

Reply:
The question is for 'pathlib' module, not for 'shutil'